src/js/owl.carousel.js

```diff
@@ -9,7 +9,17 @@
  * @todo Test Zepto
  * @todo stagePadding calculate wrong active classes
  */
-;(function($, window, document, undefined) {
+!(function(root, factory) {
+	if (typeof define === 'function' && define.amd) {
+		define(['jquery'], function($) {
+			return factory(root, $);
+		});
+	} else if (typeof exports === 'object') {
+		factory(root, require('jquery'));
```

Comment:
Would be nice if we could check if jQuery already is loaded here.
```
window.jQuery || require('jquery')
```
Because in some cases, like in Drupal jQuery is loaded by default.